lib/foreman_statistics/engine.rb

```diff
@@ -10,6 +10,7 @@ class Engine < ::Rails::Engine
 
     # Add any db migrations
     initializer 'foreman_statistics.load_app_instance_data' do |app|
+      next unless Gem::Dependency.new('', '>= 2.2').match?('', SETTINGS[:version])
```

Comment:
Should we? I think the migrations wouldn't run again since they use the same identifier, but if there's some namespacing at play here we should collapse them to one migration and run it defensively (i.e. `unless table_exists?...` etc)

Reply:
They wouldn't run again. But if both are present, Rails complain about duplicate migration.

Reply:
should we collapse them then anyways to a new one?

Reply:
usure here :thinking: we would be left with `schema_migration` record of them in database, I don't like that and I'm not sure what could we do about that.